outlines/blaze.md

```diff
@@ -0,0 +1,58 @@
+# Blaze
+
+1. Introduction to Spacebars -- the tracker-backed handlebars like templating syntax
+  1. Example of spacebars syntax, data context + helpers
+  2. Data contexts and access - name lookup (`.`, `this`, `..`, `[0]`), and null values
+  3. Helpers, arguments, options
+  4. Inclusion of other templates + arguments, passing data contexts
+  5. Helpers inside tags -- returning strings and objects, `checked={{isChecked}}`
+  6. Nested helpers + sub expressions
+  7. Block helpers (you can create them with templates, see 10. below)
+  8. Safestrings and `{{{`
+  9. Builtin block helpers
+    1. `{{#if/unless}}`
+    2. `{{#each .. in ..}}`
+    3. `{{#let}}`
+      1. NOTE: we need to ensure that issues around lexical scope and event handlers are resolved before we fail to even mention `{{#each}}` and `{{#with}}`. But we should attempt it.
```

Comment:
> I think in general, in the guide we mention things we think people should use.

Yea, but I think that with proper use they are still useful. So maybe we should document proper use, instead of not documenting it at all. :-)